### ISSUE TITLE
Bug: to_td can't handle preallocated column as time column

### ISSUE THREAD

**Issue by kevhill** [treasure-data/pandas-td#3]
If you rename a column in pandas you can't seem to access it correctly in the `time_col` parameter.

```
>>> df_renamed = pd.DataFrame(index=range(4), columns=('foo','to_be_time'))
>>> for i in range(4):
...     df_renamed['foo'][i] = i
...     df_renamed['to_be_time'][i]=10000*i
...
>>> print df_renamed
  foo to_be_time
0   0          0
1   1      10000
2   2      20000
3   3      30000
>>> df_renamed.rename(columns={'to_be_time':'time'}, inplace=True)
>>> print df_renamed
  foo   time
0   0      0
1   1  10000
2   2  20000
3   3  30000
>>> pandas_td.to_td(df_renamed, 'kh_dev.test', td_conn, if_exists='replace', time_col='time', index=False)
>>> sleep(10)
>>> pandas_td.read_td_table('test', engine)
   foo  time
0    0     0
1    1     0
2    2     0
3    3     0
```
Other renamed column seem to be fine

**Comment by kevhill** [comment]
Update: looks like it is just the preallocation step

```
>>> df_renamed = pd.DataFrame(index=range(4), columns=('foo','to_be_time'))
>>> for i in range(4):
...     df_renamed['foo'][i] = i
...     df_renamed['to_be_time'][i]=10000*i
...
>>> print df_renamed
  foo to_be_time
0   0          0
1   1      10000
2   2      20000
3   3      30000
>>>
>>> pandas_td.to_td(df_renamed, 'kh_dev.test', td_conn, if_exists='replace', time_col='to_be_time', index=False)
>>> sleep(10)
>>> pandas_td.read_td_table('test', engine)
   foo  time
0    0     0
1    1     0
2    2     0
3    3     0
```

**Comment by k24d** [comment]
Reproduced on my side.  Let me investigate.

**Comment by k24d** [comment]
I confirmed this is a bug and will fix it soon.

Currently `to_td()` converts time values depending on its type.  It seems the type of your time column was `dtype('O')` when initially created, in which case `to_td()` doesn't work as expected:

```python
In [1]: df = pd.DataFrame(index=range(4), columns=('foo','to_be_time'))

In [2]: df
Out[2]:
   foo to_be_time
0  NaN        NaN
1  NaN        NaN
2  NaN        NaN
3  NaN        NaN

In [3]: df['to_be_time'].dtype
Out[3]: dtype('O')
```

**Comment by k24d** [comment]
released v0.8.7
